contracts/bonding/BondingManager.sol

```diff
@@ -1425,4 +1469,34 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     function roundsManager() internal view returns (IRoundsManager) {
         return IRoundsManager(controller.getContract(keccak256("RoundsManager")));
     }
+
+    function _onlyTicketBroker() internal {
```

Comment:
All of these internal functions besides `_autoClaimEarnings()` can be marked with `view`.

Reply:
fixed in e3284a4